### ISSUE TITLE
get error when building source code on Red Hat 8.1

### ISSUE THREAD

**Issue by amaneru55** [milvus-io/milvus#4458]
I have tried to build the source code in red hat 8.1 after install all denpendences written in file “/core/centos7_build_deps.sh”, but i got flowing error message.

````shell
[100%] Built target xccblat3
[100%] Linking Fortran executable xzcblat3
[100%] Built target xzcblat3
[ 27%] Performing install step for 'openblas_ep'
make[3]: warning: jobserver unavailable: using -j1.  Add '+' to parent make rule.
[  4%] Built target kernel_EXCAVATOR
[  8%] Built target interface
[ 11%] Built target driver_level2
[ 14%] Built target driver_level3
[ 14%] Built target driver_others
[ 14%] Built target lapack
[ 18%] Built target kernel_ZEN
[ 21%] Built target kernel_HASWELL
[ 25%] Built target kernel_SKYLAKEX
[ 29%] Built target kernel_OPTERON
[ 33%] Built target kernel_DUNNINGTON
[ 37%] Built target kernel_NEHALEM
[ 41%] Built target kernel_PENRYN
[ 45%] Built target kernel_OPTERON_SSE3
[ 49%] Built target kernel_PRESCOTT
[ 53%] Built target kernel_CORE2
[ 57%] Built target kernel_NANO
[ 61%] Built target kernel_ATOM
[ 65%] Built target kernel_BOBCAT
[ 68%] Built target kernel_BARCELONA
[ 72%] Built target kernel_SANDYBRIDGE
[ 76%] Built target kernel_BULLDOZER
[ 79%] Built target kernel_PILEDRIVER
[ 83%] Built target kernel_STEAMROLLER
[ 99%] Built target openblas
[ 99%] Built target openblas_utest
[ 99%] Built target sblat3
[ 99%] Built target dblat3
[ 99%] Built target cblat1
[ 99%] Built target dblat1
[ 99%] Built target sblat1
[ 99%] Built target zblat2
[ 99%] Built target dblat2
[ 99%] Built target cblat2
[ 99%] Built target sblat2
[ 99%] Built target zblat1
[100%] Built target zblat3
[100%] Built target cblat3
[100%] Built target xdcblat2
[100%] Built target xdcblat1
[100%] Built target xccblat2
[100%] Built target xccblat1
[100%] Built target xdcblat3
[100%] Built target xscblat3
[100%] Built target xscblat2
[100%] Built target xscblat1
[100%] Built target xccblat3
[100%] Built target xzcblat1
[100%] Built target xzcblat2
[100%] Built target xzcblat3
Install the project...
-- Install configuration: "Release"
-- Installing: /home/project/milvus/core/cmake_build/src/index/openblas_ep-prefix/src/openblas_ep/include/openblas/openblas_config.h
-- Installing: /home/project/milvus/core/cmake_build/src/index/openblas_ep-prefix/src/openblas_ep/include/openblas/f77blas.h
-- Installing: /home/project/milvus/core/cmake_build/src/index/openblas_ep-prefix/src/openblas_ep/include/openblas/cblas.h
-- Installing: /home/project/milvus/core/cmake_build/src/index/openblas_ep-prefix/src/openblas_ep/lib/pkgconfig/openblas.pc
-- Installing: /home/project/milvus/core/cmake_build/src/index/openblas_ep-prefix/src/openblas_ep/share/cmake/OpenBLAS/OpenBLASConfig.cmake
-- Installing: /home/project/milvus/core/cmake_build/src/index/openblas_ep-prefix/src/openblas_ep/share/cmake/OpenBLAS/OpenBLASConfigVersion.cmake
-- Installing: /home/project/milvus/core/cmake_build/src/index/openblas_ep-prefix/src/openblas_ep/share/cmake/OpenBLAS/OpenBLASTargets.cmake
-- Installing: /home/project/milvus/core/cmake_build/src/index/openblas_ep-prefix/src/openblas_ep/share/cmake/OpenBLAS/OpenBLASTargets-release.cmake
[ 28%] Completed 'openblas_ep'
[ 28%] Built target openblas_ep
make: *** [Makefile:130: all] Error 2
[root@iZrj985wwblwfyh84whg1rZ core]#
````
the flowing shell scirpt is what I have installed in my OS
````shell
sudo mkdir -p /home/source && cd /home/source && \
sudo wget "https://cmake.org/files/v3.14/cmake-3.14.3-Linux-x86_64.tar.gz" && \
sudo tar -zxvf cmake-3.14.3-Linux-x86_64.tar.gz --strip-components=1  -C /usr/local &&\
sudo yum install -y  make automake python3-pip libcurl-devel python3-devel  \
mysql-devel gcc gcc-c++ boost clang clang-tools-extra openssl-devel \
gcc-gfortran lapack git boost-devel && \
sudo wget "http://mirror.centos.org/centos/8/PowerTools/x86_64/os/Packages/boost-static-1.66.0-10.el8.x86_64.rpm" && \
sudo rpm -ivh ./boost-static-1.66.0-10.el8.x86_64.rpm && \
sudo wget "https://github.com/ccache/ccache/releases/download/v3.6/ccache-3.6.tar.xz" && \
sudo tar -xvf ccache-3.6.tar.xz && cd ccache-3.6 && \
sudo ./configure -prefix=/var/ccache && \
sudo make -j8 && make install && \
sudo ln -s /var/ccache/bin/ccache /usr/bin/ccache && \
sudo cd /home/source && \
sudo wget "https://github.com/linux-test-project/lcov/releases/download/v1.15/lcov-1.15.tar.gz" && \
sudo tar -zxvf lcov-1.15.tar.gz && cd lcov-1.15 && \
sudo make install  && \
sudo cd /home/source && \
sudo wget "http://rpmfind.net/linux/centos/8-stream/PowerTools/x86_64/os/Packages/blas-devel-3.8.0-8.el8.x86_64.rpm" && \
sudo wget "http://mirror.centos.org/centos/8/PowerTools/x86_64/os/Packages/lapack-devel-3.8.0-8.el8.x86_64.rpm" && \
sudo rpm -ivh ./blas-devel-3.8.0-8.el8.x86_64.rpm && \
sudo rpm -ivh ./lapack-devel-3.8.0-8.el8.x86_64.rpm && \
sudo mkdir -p /home/project && cd /home/project && \
sudo git clone https://github.com/milvus-io/milvus.git --branch 0.10.0 && \
sudo /home/project/milvus/core/build.sh -t Release
````
OS: Red hat 8.1
milvus: 0.10.0

If there any depencedences i should install?

**Comment by yhmo** [comment]
We didn't try compile milvus on Red Hat. There are two scripts under milvus/core folder:
centos7_build_deps.sh and ubuntu_build_deps.sh
Most of dependencies are listed in the script.

**Comment by hyyz17200** [comment]
Me too,  ubuntu 20.04,  milvus 0.10.0.

**Comment by fooSynaptic** [comment]
@shengjun1985 for comments above i didn't see this issue was solved which in Red hat or on ubuntu, i was facing same issue on ubuntu, so why this issue closed.

**Comment by shengjun1985** [comment]
Sorry, because we have no resource and plan to fix the issues about Red Hat 8.1.
